### PR TITLE
Components Created: Insights, Portfolio & Testimonials.

### PR DESCRIPTION
**Pull request by minhaj-ops**

### CODE REVIEW

File: src/app/page.jsx
Comment (DevPro15) [inline src/app/page.jsx:1]: Use box or `stack` instead of `div`

File: src/components/header/header.jsx
Comment (DevPro15) [inline src/components/header/header.jsx:1]: Try to avoid else if in ternary condition create a separate function to handle it .

**Review by DevPro15 — COMMENTED** [review]
Please Review the comments and update them.

File: src/app/page.jsx
Comment (minhaj-ops) [reply src/app/page.jsx:1]: Okay

File: src/components/header/header.jsx
Comment (minhaj-ops) [reply src/components/header/header.jsx:1]: const isActive = (sectionName) => {
    return activeSection === sectionName ? true : false;
  };
  
  in line 196 i have created that function to manage the isActive thing.

**Pull request merged**